x-pack/plugins/ml/public/application/routing/routes/trained_models/nodes_list.tsx

```diff
@@ -23,6 +23,9 @@ export const nodesListRouteFactory = (
 ): MlRoute => ({
   path: '/trained_models/nodes',
   render: (props, deps) => <PageWrapper {...props} deps={deps} />,
+  title: i18n.translate('xpack.ml.modelManagement.nodesOverview.docTitle', {
+    defaultMessage: 'Nodes Overview',
```

Comment:
Is "overview" required here? Is it supposed to imply some difference from the type of functionality you get on the Trained Models page, for example? If not, I recommend omitting the unnecessary word:
```suggestion
    defaultMessage: 'Nodes',
```

Reply:
Agree it should just be `Nodes`, which would be consistent with the changes in #124049.